renv.lock

```diff
@@ -41,7 +35,7 @@
       "Package": "EpiNow2",
       "Version": "1.3.6.7000",
       "Source": "Repository",
-      "Repository": "https://epiforecasts.r-universe.dev",
+      "RemoteType": "repository",
```

Comment:
Other than this one these are all just because your not on Linux. 

This one is more of a problem because we want to install EpiNow2 from it's universe Vs from GitHub because then windows and Mac users get precompiled binaries. 

If you could revert this one that would be great.

Reply:
(I think you've introduced but not entirely clear to me.)

Reply:
Ah this was an accident -- I was just trying to add bookdown so that the report could be knit. Fixed.